modules/navier_stokes/tests/ins/lid_driven/lid_driven_supg.cmp

```diff
@@ -0,0 +1,10 @@
+COORDINATES absolute 1.e-6    # min separation not calculated
+
+TIME STEPS relative 1.e-6 floor 0.0     # min:               0 @ t1 max:             2.5 @ t6
+
+# No GLOBAL VARIABLES
+
+NODAL VARIABLES relative 1.e-6 floor 1.e-12
+	p      absolute 5.e-4
```

Comment:
Seems like a very loose tolerance for pressure? Is this just copied from the other lid-driven custom comparison file?